docs/design_decisions.md

```diff
@@ -113,7 +113,7 @@ Volgens LO GBA zijn ook relaties die zijn beëindigd nog "actueel". Voor het zoe
 Wanneer wel behoefte is aan alle partner-relaties, inclusief relaties die al beëindigd zijn, kan hiervoor historisch zoeken op partners worden gebruikt.
 
 ## Opvragen van historie kan via specifieke endpoints
-Opvragen van historie wordt gedefinieerd met aparte endpoints. Als een provider ervoor kiest om deze niet te implementeren wordt er een 501 geretourneerd. De historische bevraging krijgen dus resourcenaam waaruit blijkt dart het om een historische resource gaat. 
+Opvragen van historie wordt gedefinieerd met aparte endpoints. Als een provider ervoor kiest om deze niet te implementeren wordt er een 501 geretourneerd. De historische bevraging krijgen dus resourcenaam waaruit blijkt dart het om een historische resource gaat.
```

Comment:
> De historische bevraging krijgen dus resourcenaam waaruit blijkt dart het om een historische resource gaat.

corrigeren in:

_De historische bevragingen krijgen dus resourcenamen waaruit blijkt dat het om historische resources gaat._

Reply:
Aangepast

Reply:
Je hebt niet alle correcties aangebracht:

resourcenaam > resourcenamen 
dart > dat
resource > resources

Aanpassing van de volgorde van de zinnen heb je ook niet doorgevoerd maar ik ga er vanuit dat dit een bewuste keuze is.